lib/scope.js

```diff
@@ -110,15 +114,11 @@ AST_Toplevel.DEFMETHOD("figure_out_scope", function(options){
     var in_destructuring = null;
     var in_export;
     var tw = new TreeWalker(function(node, descend){
-        var create_a_block_scope =
-            (options.screw_ie8 && node instanceof AST_Catch) ||
```

Comment:
I'm pretty sure there might be something broke in https://github.com/mishoo/UglifyJS2/commit/634f231b78853dd0f02f6857eb9d4a646d571029. But meh, I didn't take the time yet to include some tests.